compass/landice/mesh.py

```diff
@@ -102,3 +102,103 @@ def set_rectangular_geom_points_and_edges(xmin, xmax, ymin, ymax):
         dtype=jigsawpy.jigsaw_msh_t.EDGE2_t)
 
     return geom_points, geom_edges
+
+
+def set_cell_width(self, section, thk, vx=None, vy=None,
+                   dist_to_edge=None, dist_to_grounding_line=None):
+    """
+    Set cell widths based on settings in config file to pass to
+    :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
+    Requires the following options to be set in the given config section:
+    ``min_spac``, ``max_spac``, ``high_log_speed``, ``low_log_speed``,
+    ``high_dist``, ``low_dist``, and ``cull_distance``.
```

Comment:
It looks like `use_speed`, `use_dist_to_grounding_line` and `use_dist_to_edge` are also required, right?

Reply:
Oops, yes. Apologies for sloppiness. This and suggestions below are addressed in https://github.com/MPAS-Dev/compass/pull/321/commits/ff7525d211d68e13a93f233841235422f0b35bce.